sdk/storage/Azure.Storage.Blobs/src/Models/BlobDownloadOptions.cs

```diff
@@ -1,87 +1,88 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// TODO #27253
+using System;
+using Azure.Core;
 
-//using System;
-//using Azure.Core;
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Optional parameters for downloading a range of a blob.
+    /// </summary>
+    public class BlobDownloadOptions
+    {
+        /// <summary>
+        /// If provided, only download the bytes of the blob in the specified
+        /// range.  If not provided, download the entire blob.
+        /// </summary>
+        public HttpRange Range { get; set; }
 
-//namespace Azure.Storage.Blobs.Models
-//{
-//    /// <summary>
-//    /// Optional parameters for downloading a range of a blob.
-//    /// </summary>
-//    public class BlobDownloadOptions
-//    {
-//        /// <summary>
-//        /// If provided, only download the bytes of the blob in the specified
-//        /// range.  If not provided, download the entire blob.
-//        /// </summary>
-//        public HttpRange Range { get; set; }
+        /// <summary>
+        /// Optional <see cref="BlobRequestConditions"/> to add conditions on
+        /// downloading this blob.
+        /// </summary>
+        public BlobRequestConditions Conditions { get; set; }
 
-//        /// <summary>
-//        /// Optional <see cref="BlobRequestConditions"/> to add conditions on
-//        /// downloading this blob.
-//        /// </summary>
-//        public BlobRequestConditions Conditions { get; set; }
+        /// <summary>
+        /// Optional <see cref="IProgress{Long}"/> to provide
+        /// progress updates about data transfers.
+        /// </summary>
+        public IProgress<long> ProgressHandler { get; set; }
 
-//        /// <summary>
-//        /// Optional <see cref="IProgress{Long}"/> to provide
-//        /// progress updates about data transfers.
-//        /// </summary>
-//        public IProgress<long> ProgressHandler { get; set; }
+        /// <summary>
+        /// Options for transfer validation settings on this operation.
+        /// When transfer validation options are set in the client, setting this parameter
+        /// acts as an override.
+        /// Set <see cref="DownloadTransferValidationOptions.Validate"/> to false if you
+        /// would like to skip SDK checksum validation and validate the checksum found
+        /// in the <see cref="Response"/> object yourself.
+        /// Range must be provided explicitly, stating a range withing Azure
+        /// Storage size limits for requesting a transactional hash. See the
+        /// <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob">
+        /// REST documentation</a> for range limitation details.
+        /// </summary>
+        public DownloadTransferValidationOptions TransferValidationOptions { get; set; }
 
+        /// <summary>
+        /// Default constructor.
+        /// </summary>
+        public BlobDownloadOptions()
```

Comment:
Nit: I don't think we need this constructor.

Reply:
Hmm. There is a private copy constructor in place but I'm not sure what it was doing back when it was made. Will investigate.